sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -7165,9 +7165,9 @@
                   "tags": [],
                   "properties": {},
                   "conditional": {
-                    "show": null,
-                    "when": null,
-                    "eq": "",
```

Comment:
Why is this updated?

And also the content changes are pertaining to protected person. 

![image](https://github.com/user-attachments/assets/6853f22e-08fa-47d6-b47b-0658f5d10676)